portal-web/test/functional/com/liferay/portalweb/tests/mytests/Form/Form.testcase

```diff
@@ -0,0 +1,129 @@
+@component-name = "portal-web-forms-and-data-lists"
+definition {
+
+	property testray.main.component.name = "Forms";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+	}
+
+	tearDown {
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content & Data",
+			portlet = "Forms");
+
+		FormsAdmin.viewForm(
+			formDescription = "English Description",
+			formName = "English Form");
+
+		LexiconEntry.gotoVerticalEllipsisMenuItemNoError(menuItem = "Delete");
+
+		AssertConfirm(value1 = "Are you sure you want to delete this? It will be deleted immediately.");
+
+		Alert.viewSuccessMessage();
+
+		User.logoutAndLoginPG(
+			userLoginEmailAddress = "test@liferay.com",
+			userLoginFullName = "Test Test");
+	}
+
+	@description = "Access Portal and create e validate NEW FORMS"
+	@priority = "5"
+	test ValidateFormsPortal {
```

Comment:
@victorhcunha it would be interesting to apply BDD in this test, separating macros on tasks with given-when-then. What do you think?

Reply:
Updated with BDD tasks